library/stringscanner/scan_until_spec.rb

```diff
@@ -7,9 +7,9 @@
   end
 
   it "returns the substring up to and including the end of the match" do
-   @s.scan_until(/a/).should == "This is a"
-   @s.pre_match.should == "This is "
-   @s.post_match.should == " test"
+    @s.scan_until(/a/).should == "This is a"
+    @s.pre_match.should == "This is "
+    @s.post_match.should == " test"
   end
```

Comment:
👍 
Probably there is also incorrect indentation in one of the recently merged PRs (I overlooked this as well)
https://github.com/ruby/spec/pull/1133/files#diff-2fb24e87cf5ec3cb134dd62a7631fd4db5efc36bd5857548dfba4716a43618de

Reply:
You're right, fixed that one in  #1177

Reply:
I wonder if we could check this easily with RuboCop, it might be worth a try to enable https://docs.rubocop.org/rubocop/cops_layout.html#layoutindentationconsistency